lib/jekyll-github-metadata/client.rb

```diff
@@ -68,6 +68,15 @@ def method_missing(method_name, *args, &block)
         end
       end
 
+      # NOTE: Faraday's error classes has been promoted to under Faraday module from v1.0.0.
+      # This patch aims to prevent on locking specific version of Faraday.
+      FARADAY_FAILED_CONNECTION =
+        begin
+          Faraday::Error::ConnectionFailed
+        rescue NameError
+          Faraday::ConnectionFailed
+        end
```

Comment:
@parkr this tries to detect whether `Faraday::Error::ConnectionFailed` works or not. This would prevents from locking specific version of Faraday. All tests passes for the both versions v0.17 and v1.0 of Faraday: [Build #517 - jekyll/github-metadata - Travis CI](https://travis-ci.org/jekyll/github-metadata/builds/633133752).